src/components/upload/ImageUpload.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {
+  Ref,
+  useImperativeHandle, useState,
+} from 'react';
+import { useDropzone } from 'react-dropzone';
+import s from './ImageUpload.module.scss';
+
+export interface Props {
+  onChange: (images: File[]) => void
+}
+
+interface ImageUploadRef {
+  resetState(): void;
+}
+
+export const ImageUpload = React.forwardRef<ImageUploadRef, Props>((props: Props, ref: Ref<ImageUploadRef>) => {
+  const [previews, setPreviews] = useState<any[]>([]);
```

Comment:
`previews` is typed as `any`

Reply:
good catch, I've created a type for the previews.